src/otx/core/metrics/average_precision_3d.py

```diff
@@ -32,6 +34,7 @@ def __init__(
         super().__init__()
 
         self.label_info: LabelInfo = label_info
+        self.mean_ap: MeanAveragePrecision = MeanAveragePrecision(box_format="xyxy", iou_type="bbox")
```

Comment:
How about adding a code to reset `mean_ap` in `reset` function?

Reply:
Added https://github.com/openvinotoolkit/training_extensions/pull/4031/commits/779cc8f3c3e8682d264b56f2344ea2e5f3a5a70a